crates/oxc_minifier/src/ast_passes/peephole_replace_known_methods.rs

```diff
@@ -217,7 +220,7 @@ mod test {
         fold_same("x = 'abcdef'.indexOf([1,2])");
 
         // Template Strings
-        fold_same("x = `abcdef`.indexOf('b')");
+        fold("x = `abcdef`.indexOf('b')", "x = 1;");
```

Comment:
Our test infra are ported from Google Closure Compiler, where it does not fold the template strings, and related code here: https://github.com/google/closure-compiler/blob/master/test/com/google/javascript/jscomp/PeepholeReplaceKnownMethodsTest.java#L114.

@Boshen any suggestions?

Reply:
@shulaoda The template string contains some expressions, but some of them can't be handled. There are already methods in place to replace template strings with normal strings when possible, and the minification process iterates multiple times rather than being a single pass. So, I believe we can focus on the string itself and not worry about template strings here. Everything else looks fantastic—thanks for your contribution!

Reply:
So don't worry these tests here - after several times, from the template string, to the normal string, the minifier can handle this scenario :-)

Reply:
You are correct, I saw that the `closure-compiler` did recognize and convert the `TemplateLiteral`.